tasks/utilities/data_handling/task_arln_stats.wdl

```diff
@@ -0,0 +1,80 @@
+version 1.0
+
+task arln_stats {
+    input {
+        String samplename
+        String taxon
+        String workflow_type
+
+        Int genome_length
+        File? read1_raw
+        File? read2_raw
+        File? read1_clean
+        File? read2_clean
+
+        Int cpu = 2
+        Int memory = 5
+        Int disk_size = 10
+        String docker = "us-docker.pkg.dev/general-theiagen/theiagen/arln_stats:1.0.0"
+    }
+    
+    command <<<
+        set -euo pipefail
+
+        # Generate Q30 statistics raw reads
+        if [[ ~{workflow_type} == "pe" ]]; then
+            python /scripts/q30.py -i ~{read1_raw} ~{read2_raw} > RAW_Q30
+            awk '/read1/ {printf "%.2f\n", $2}' RAW_Q30 > read1_raw_q30
+            awk '/read2/ {printf "%.2f\n", $2}' RAW_Q30 > read2_raw_q30
+            # Generate Q30 statistics cleaned reads
+            python /scripts/q30.py -i ~{read1_clean} ~{read2_clean} > CLEAN_Q30
+            awk '/read1/ {printf "%.2f\n", $2}' CLEAN_Q30 > read1_clean_q30
+            awk '/read2/ {printf "%.2f\n", $2}' CLEAN_Q30 > read2_clean_q30
+        elif [[ ~{workflow_type} == "se" || ~{workflow_type} == "ont" ]]; then
+            python /scripts/q30.py -i ~{read1_raw} > RAW_Q30
+            awk '/read1/ {printf "%.2f\n", $2}' RAW_Q30 > read1_raw_q30
+            echo "0.0" > read2_raw_q30
+            python /scripts/q30.py -i ~{read1_clean} > CLEAN_Q30
+            awk '/read1/ {printf "%.2f\n", $2}' CLEAN_Q30 > read1_clean_q30
+            echo "0.0" > read2_clean_q30
+        else
```

Comment:
instead of this, you could just touch these files since they're not going to be used in the outputs of FASTA

Reply:
i think a blank value is more accurate than saying 0.0